codes/1707_cpp_wodnjs.cpp

```diff
@@ -0,0 +1,68 @@
+#include<iostream>
+#include<vector>
+#include<queue>
+using namespace std;
+int vis[20001];
+void init_vis(int vtx);
+bool bip_graph(int vtx,vector<int> v[]);
+int main() {
+
+	int tcase;
+	cin >> tcase;
+	for (int i = 0; i < tcase; i++)
+	{
+		int vtx, edge;
+		vector<int> v[20001];
+		cin >> vtx >> edge;
+		init_vis(vtx);
+		for (int j = 0; j < edge; j++)
+		{
+			int from, to;
+			cin >> from >> to;
+			v[from].push_back(to);
+			v[to].push_back(from);
+		}
+		bool pass=true;
+		for (int k = 1; k <=vtx ; k++)
+		{
+			if (!bip_graph(k, v)) {
+				cout << "NO\n";
+				pass = false;
+				break;
+			}
+		}
+		if(pass)
+			cout<<"YES\n";
```

Comment:
지금은 코드가 짧아서 상관없지만 나중에 코드가 길어지면 다음과 같이 출력을 모아주는 게 가독성이 더 높을 것 같습니다.
```suggestion
			if (!bip_graph(k, v)) {
				pass = false;
				break;
			}
		}
		if(pass)
			cout<<"YES\n";
	        else
	                cout<<"NO\n";
```